frontend/src/stores/ui.ts

```diff
@@ -53,22 +62,28 @@ export interface UiState {
   /* A list of sensitive single result UUIDs the user has opted-into seeing */
   revealedSensitiveResults: string[]
   headerHeight: number
+
+  /* Whether the UI is using dark, light, or system mode */
+  colorMode: ColorMode
 }
 
 export const breakpoints = Object.keys(ALL_SCREEN_SIZES)
 
+export const defaultUiState: UiState = {
+  instructionsSnackbarState: "not_shown",
+  innerFilterVisible: false,
+  isFilterDismissed: false,
+  isDesktopLayout: false,
+  breakpoint: "sm",
+  dismissedBanners: [],
+  shouldBlurSensitive: true,
+  revealedSensitiveResults: [],
+  headerHeight: 80,
+  colorMode: "system",
+}
+
 export const useUiStore = defineStore("ui", {
-  state: (): UiState => ({
-    instructionsSnackbarState: "not_shown",
-    innerFilterVisible: false,
-    isFilterDismissed: false,
-    isDesktopLayout: false,
-    breakpoint: "sm",
-    dismissedBanners: [],
-    shouldBlurSensitive: true,
-    revealedSensitiveResults: [],
-    headerHeight: 80,
-  }),
+  state: (): UiState => ({ ...defaultUiState }),
```

Comment:
The spread here isn't _super_ necessary, but fixes an issue with tests wherin the imported defaultUiState was being mutated between tests. 

Logically this makes sense anyway, that the store would "copy in" all the properties of the default state object